python_cross_site_scripting/HelloWorldWinning_vps/u8.py

```
@@ -0,0 +1,96 @@
+import os
+from flask import Flask, request, redirect, render_template_string, Response
+from werkzeug.utils import secure_filename
+from functools import wraps
+
+app = Flask(__name__)
+
+# Default upload directory and credentials
+DEFAULT_UPLOAD_FOLDER = '/data/upload_folder'
+USERNAME = 'admin'
+PASSWORD = 'password'
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "PASSWORD"

(S105)